File: src/main/java/com/handwoong/everyonewaiter/EveryoneWaiterApplication.java
package com.handwoong.everyonewaiter;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@SpringBootApplication
public class EveryoneWaiterApplication {

    public static void main(String[] args) {
        SpringApplication.run(EveryoneWaiterApplication.class, args);
    }

}

File: src/main/kotlin/com/handwoong/everyonewaiter/EveryoneWaiterApplication.kt
package com.handwoong.everyonewaiter

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@EnableJpaAuditing
@SpringBootApplication
class EveryoneWaiterApplication

fun main(args: Array<String>) {
    runApplication<EveryoneWaiterApplication>(*args)
}
